QuestionSets/PythonLanguage/ClassesAndObjects/Naman_SolutionSet2/Class_And_Objects_SolutionSet2Q1+Q2.py

```diff
@@ -0,0 +1,172 @@
+# OOP Classes and Objects Solution_Set2 Naman Modani Q1+2
+
+class Player:
+
+    def __init__ (self):
+        age = int()
+        movement_speed = int()
+        health = int()
+        money = int()
+
+        self.age = age
+        self.movement_speed = movement_speed
+        self.health = health
+        self.money = money
+
+    def move(self):
+        self.movement_speed += 5
+
+    def stop(self):
+        self.movement_speed = 0
+
+    def take_damage(self):
+        self.health -= 5
+
+    def heal(self):
+        self.health += 5
+
+    def make_money(self):
+        self.money += 10000
+
+    def spend_money(self):
+        self.money -= 10000
+
+    def __str__ (self):
+        print("Age:", self.age)
+        print("Speed:", self.movement_speed)
+        print("Health is:", self.health)
+        print("Money is: $", self.money)
+
+BigPapaPyscho = Player()
+
+BigPapaPyscho.money = 1000000
+BigPapaPyscho.health = 100
+BigPapaPyscho.movement_speed = 0
+BigPapaPyscho.age = 3625
+
+# He's old,why not?
+
+print("----------------------")
+
+print("Move")
+BigPapaPyscho.move()
+BigPapaPyscho.__str__()
+print("\n")
+
+
+print("Stop")
+BigPapaPyscho.stop()
+BigPapaPyscho.__str__()
+print("\n")
+
+
+print("Take Damage")
+BigPapaPyscho.take_damage()
+BigPapaPyscho.__str__()
+print("\n")
+
+
+print("Heal")
+BigPapaPyscho.heal()
+BigPapaPyscho.__str__()
+print("\n")
+
+
+print("Make Money")
+BigPapaPyscho.make_money()
+BigPapaPyscho.__str__()
+print("\n")
+
+
+print("Spend Money")
+BigPapaPyscho.spend_money()
+BigPapaPyscho.__str__()
+print("\n")
+
+class Enemy:
+     
+    def __init__ (self):
+        age = int()
+        movement_speed = int()
+        health = int()
+        attack_power = int()
+
+        self.age = age
+        self.movement_speed = movement_speed
+        self.health = health
+        self.attack_power = attack_power
+
+    def move(self):
+        self.movement_speed += 5
+
+    def stop(self):
+        self.movement_speed = 0
+
+    def take_damage(self):
+        self.health -= 5
+
+    def heal(self):
+        self.health += 5
+
+    def booster(self):
+        self.attack_power += 5
+
+    def attack(self,Player):
+        Player.take_damage      
```

Comment:
@IronicalCoder @paramkpr ye kya bhai??

Reply:
@ViRu-ThE-ViRuS Sorry. He told me that it was working... Not listening to @IronicalCoder next time. My bad.